packages/components/src/SignConstitutionButton.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+import { SignConstitutionButton } from "./SignConstitutionButton";
+import { Civil } from "@joincivil/core";
+
+let civil: Civil | undefined;
+
+try {
+  civil = new Civil();
```

Comment:
Wasn't there a civil provider pattern somewhere in dapp?

Reply:
this isn't dapp, its patterns